crates/biome_lsp/src/session.rs

```diff
@@ -667,7 +662,6 @@ impl Session {
     pub(crate) async fn scan_project_folder(
```

Comment:
Since we no longer pass any folders, I would rename the method to `scan_project`.